### PR TITLE
Pay 5797

### PR DESCRIPTION
**Pull request by anshikanigam15**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by anoojhmcts — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -58,4 +58,10 @@ variable "fr_product" {
   default = "fees-register"
 }
 
-variable "aks_subscription_id" {}
\ No newline at end of file
+variable "sku" {
+  type        = string
+  default     = "Premium"
```
Comment (thomast1906) [inline variables.tf:63]: Do you want Premium SKU in all envs? For cost saving, I would only have Premium in Prod/AAT

**Review by thomast1906 — CHANGES_REQUESTED** [review]
`sku=standard` is not needed, as you have set it as default in the variable

https://github.com/hmcts/ccpayfr-shared-infrastructure/pull/80/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR63

**Comment by anshikanigam15** [comment]
> `sku=standard` is not needed, as you have set it as default in the variable
> 
> see here : https://github.com/hmcts/ccpayfr-shared-infrastructure/pull/80/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR63

Removed SKU Standard

**Review by thomast1906 — APPROVED** [review]
Approving for now, depending you may want likes of AAT/perftest to be same as Production.

Feel free to try this to see how it is.

**Review by thomast1906 — CHANGES_REQUESTED** [review]
Lets hold on this, to discuss which envs we want as premium, I do think perrftest, aat, prod

**Comment by thomast1906** [comment]
perftest & AAT you probably want to be Premium as well to align PTL environments

**Comment by anshikanigam15** [comment]
> perftest & AAT you probably want to be Premium as well to align PTL environments



> 

Thanks made both of them as premium

**Pull request merged**